operate/cli.py

```diff
@@ -936,6 +943,136 @@ def _daemon(
     server.run()
 
 
+@_operate.command(name="quickstart")
+def qs_start(
+    config: Annotated[str, params.String(help="Quickstart config file path")],
+) -> None:
+    """Quickstart."""
+    operate = OperateApp()
+    operate.setup()
+    run_service(operate=operate, config_path=config)
+
+
+@_operate.command(name="quickstop")
```

Comment:
`quickstop` sounds strange.

Reply:
I'm bad at naming ':D
Could you suggest?